simphony_ui/openfoam_wrapper_creation.py

```diff
@@ -109,3 +109,76 @@ def get_boundary_condition_description(bc):
             return bc.type, bc.fixed_value
     raise ValueError(
         '{} is not a possible boundary condition type'.format(bc.type))
+
+
+def create_openfoam_mesh(openfoam_wrapper, openfoam_settings):
+    """ Creates the Openfoam dataset from the settings as provided
+    by the model object
+
+    Parameters
+    ----------
+    openfoam_wrapper : Wrapper
+        The Openfoam wrapper in which you want to put the dataset
+    openfoam_settings : OpenfoamModel
+        The traited model describing the openfoam parameters
+
+    Returns
+    -------
+    mesh_dataset :
+        The dataset representing the openfoam mesh
+
+    Raises
+    ------
+    ValueError :
+        If the mesh type specified in openfoam_settings is not supported
+    """
+    if openfoam_settings.mesh_type == 'block':
+        path = openfoam_settings.output_path if \
```

Comment:
try not to have \ by using parentheses in a smart way. In general, the "pleasant way" is
```
path = (x 
        if bla bla
        else y)
```